src/core/Microsoft.PowerToys.Settings.UI/Controls/HotkeySettingsControl.xaml

```diff
@@ -8,7 +8,27 @@
     mc:Ignorable="d"
     d:DesignHeight="300"
     d:DesignWidth="400">
-    <TextBox x:Name="HotkeyTextBox" 
-             IsReadOnly="True"
-             Header="{x:Bind Header, Mode=OneTime}"/>
-</UserControl>
+    <StackPanel Orientation="Vertical">
+        <ToolTipService.ToolTip>
+            <ToolTip>
+                <StackPanel Orientation="Vertical">
+                    <TextBlock x:Uid="ShortcutWarningLabel"/>
+                    <TextBlock Text="Win, Ctrl, Alt, Shift" FontWeight="SemiBold"/>
+                </StackPanel>
+            </ToolTip>
+        </ToolTipService.ToolTip>
+
+        <StackPanel Orientation="Horizontal">
+            <local:BodyTextBlock x:Name="TitleText"
+                        Margin="0,-12,0,0"
+                        Text="{x:Bind Header, Mode=OneTime}"/>
+            <local:BodyTextBlock x:Name="TitleGlyph" Text="&#xE946;"

```

Comment:
@ryanbodrug-microsoft was suggesting we use a **TEXTBOX** instead of custom controls. Do you think we can do that here?

Reply:
Feels like overkill.. Actually, I'm wondering why if we could make a simple converter, attach that to every (normal) TextBlock (Windows Community has something out of the box for this).

We could bind the IsEnabled property to the Foreground and run it to the Converter. The Converter would then return the 'disabled brush' or the 'enabled brush'? Does that even make sense :-)?

Reply:
Good old WPF :( 
```
<Style TargetType="{x:Type TextBox}">
  <Style.Trigger>
       <Trigger Property="IsEnabled" Value="true">
     <Setter Property="Foreground" Value="myBrush">
</Trigger>
  <Style.Trigger>
</Style>
```
Anyway, here we should be able to use styles as well through VisualState right? Then we can just use this style where we need it (or make it default for all).Having custom control for such a simple thing is not very nice.
```
<Style TargetType="TextBox">
<Style Property="Template">
<Setter.Value>
<ControlTemplate TargetType="TextBox">
....
<VisualStateManager.VisualStateGroups>
<VisualStateGroup x:Name="CommonStates">
<VisualState x:Name="Disabled">
...
</VisualState>

```
and if you have some non-standard visual state you can add your own stateTrigger:
```

<VisualState>
<VisualState.StateTriggers>
<StateTrigger IsActive="{Binding Whatever,RelativeSource={RelativeSource TemplatedParent}, Mode=OneWay}"/>
```
... definition

Reply:
@martinchrzan These style setters will have to be added on every settings page and if one thing changes on that format we will have to change it on every page too. Using custom controls allows us to have this format managed in one place.

Reply:
I have now implemented the following (and it's working great:

![image](https://user-images.githubusercontent.com/9866362/88097190-15d57680-cb98-11ea-9e42-d9a2ef84f63a.png)

![image](https://user-images.githubusercontent.com/9866362/88097238-27b71980-cb98-11ea-935b-6b67d71bfc58.png)

@laviusmotileng-ms OK with this change?

Reply:
Yes that great!! Are you going to apply it to all settings that used the custom textblock control?

Reply:
Yep, will replace all custom (title)textblocks and remove those.

Reply:
@laviusmotileng-ms this is incorrect.
Guys, why are we doing this instead of using an implicit or explicit styles? This is a prime example for using styles! We have a folder called styles - add a new resource dictionary called TextBlocks (and add it into App.xml) - define the style with VisualStates and either don't set a key for it - so every textblock will behave like this or set a key and then set style here by using that key. We don't need any converters or a custom controls for changing the visual state of a simple textblock (even naming in the style definition suggest to use it ;) )

Reply:
@martinchrzan It is indeed what I would do within a e.g. DataTemplate.. But we don't have an IsEnabled property for TextBlock to bind the bool to, right? So we'd need to define a custom StateTrigger on every page and x:Bind the IsEnabled property of the module to it and then switch styles?

Maybe I'm totally missing your point - please let me know :)!

Reply:
Hm, I am again slipping into WPF where you could define your relative ancestor to search for binding, UWP does not have it :(
In this case, the only option would be to call view model by the same name such as x:Key="viewModel" - then you could bind to that element from a style, but every setting view would have to have the same name defined and I don't like this convention based approach. 
Let's keep it as you did it and I will try to dig into it later to see if we can make it nicer.

Reply:
@martinchrzan  Haha, I have that from time to time, when let's re-evaluate whenever we go to WinUI 3.0.

@laviusmotileng-ms Latest commit removes all custom textblocks and minor changes to the HotKeyControl. Please check if nothing is breaking.